lib/ansible/modules/storage/netapp/netapp_e_mgmt_interface.py

```diff
@@ -0,0 +1,708 @@
+              beginning with a value of 1.
+            - Mutually exclusive with I(channel).
+        aliases:
+            - port
+            - iface
+    channel:
+        description:
+            - The port to modify the configuration for.
+            - The channel represents the port number (typically from left to right on the controller),
+              beginning with a value of 1.
+            - Mutually exclusive with I(name).
+    address:
+        description:
+            - The IPv4 address to assign to the interface.
+            - Should be specified in xx.xx.xx.xx form.
+            - Mutually exclusive with I(config_method=dhcp)
+        required: no
+    subnet_mask:
+        description:
+            - The subnet mask to utilize for the interface.
+            - Should be specified in xx.xx.xx.xx form.
+            - Mutually exclusive with I(config_method=dhcp)
+        required: no
+    gateway:
+        description:
+            - The IPv4 gateway address to utilize for the interface.
+            - Should be specified in xx.xx.xx.xx form.
+            - Mutually exclusive with I(config_method=dhcp)
+        required: no
+    config_method:
+        description:
+            - The configuration method type to use for network interface ports.
+            - dhcp is mutually exclusive with I(address), I(subnet_mask), and I(gateway).
+        choices:
+            - dhcp
+            - static
+        required: no
+    dns_config_method:
+        description:
+            - The configuration method type to use for DNS services.
+            - dhcp is mutually exclusive with I(dns_address), and I(dns_address_backup).
+        choices:
+            - dhcp
+            - static
+        required: no
+    dns_address:
+        description:
+            - Primary IPv4 DNS server address
+        required: no
+    dns_address_backup:
+        description:
+            - Backup IPv4 DNS server address
+            - Queried when primary DNS server fails
+        required: no
+    ntp_config_method:
+        description:
+            - The configuration method type to use for NTP services.
+            - disable is mutually exclusive with I(ntp_address) and I(ntp_address_backup).
+            - dhcp is mutually exclusive with I(ntp_address) and I(ntp_address_backup).
+        choices:
+            - disable
+            - dhcp
+            - static
+        required: no
+    ntp_address:
+        description:
+            - Primary IPv4 NTP server address
+        required: no
+    ntp_address_backup:
+        description:
+            - Backup IPv4 NTP server address
+            - Queried when primary NTP server fails
+        required: no
+    ssh:
+        type: bool
+        description:
+            - Enable ssh access to the controller for debug purposes.
+            - This is a controller-level setting.
+            - Older controllers may enable rlogin or telnet instead of ssh.
```

Comment:
So this modules will do...?

Reply:
This is a bit confusing if you're not familiar with the E-Series systems. This option will enable/disable telnet or ssh, depending on the age/model of the E-Series system that's being managed. Perhaps it's worded poorly.

Reply:
Much better, thanks